example/macos/Runner/AppDelegate.swift

```diff
@@ -1,9 +1,13 @@
 import Cocoa
 import FlutterMacOS
 
-@NSApplicationMain
+@main
```

Comment:
Automated by Flutter CLI. This is not specific to SPM but this is outdated and is already in the `stable` channel.